lib/daemon.js

```diff
@@ -2,20 +2,29 @@ import readline from 'readline';
 
 import Importer from './Importer';
 
+const commandsToFunctionNames = {
+  fix: 'fixImports',
+  word: 'import',
+  goto: 'goto',
+  rewrite: 'rewriteImports',
+  add: 'addImports',
+};
+
 export default function daemon() {
   const rlInterface = readline.createInterface({
     input: process.stdin,
     output: process.stdout,
     terminal: false,
   });
 
-  rlInterface.on('line', (jsonCommand: string) => {
-    const command = JSON.parse(jsonCommand);
+  rlInterface.on('line', (jsonPayload: string) => {
+    const payload = JSON.parse(jsonPayload);
     const importer = new Importer(
-      command.fileContent.split('\n'), command.pathToFile);
+      payload.fileContent.split('\n'), payload.pathToFile);
 
     // TODO: don't automatically invoke function here because of security issues
```

Comment:
At this point it should be pretty trivial to just throw an error if `!functionName`